src/HTTPure.purs

```diff
@@ -13,143 +13,143 @@ module HTTPure
 
 import HTTPure.Body (toBuffer, toStream, toString)
 import HTTPure.Headers (Headers, empty, header, headers)
-import HTTPure.Lookup (at, (!@), has, (!?), lookup, (!!))
+import HTTPure.Lookup (at, has, lookup, (!!), (!?), (!@))
 import HTTPure.Method (Method(..))
 import HTTPure.Path (Path)
 import HTTPure.Query (Query)
 import HTTPure.Request (Request, fullPath)
 import HTTPure.Response
   ( Response
   , ResponseM
-  , response
-  , response'
-  , emptyResponse
-  , emptyResponse'
+  , accepted
+  , accepted'
+  , alreadyReported
+  , alreadyReported'
   -- 1xx
+  , badGateway
+  , badGateway'
+  , badRequest
+  , badRequest'
+  , conflict
+  , conflict'
+  -- 2xx
   , continue
   , continue'
-  , switchingProtocols
-  , switchingProtocols'
-  , processing
-  , processing'
-  -- 2xx
-  , ok
-  , ok'
   , created
   , created'
-  , accepted
-  , accepted'
-  , nonAuthoritativeInformation
-  , nonAuthoritativeInformation'
-  , noContent
-  , noContent'
-  , resetContent
-  , resetContent'
-  , partialContent
-  , partialContent'
-  , multiStatus
-  , multiStatus'
-  , alreadyReported
-  , alreadyReported'
+  , emptyResponse
+  , emptyResponse'
+  , expectationFailed
+  , expectationFailed'
+  , failedDependency
+  , failedDependency'
+  , forbidden
+  , forbidden'
+  , found
+  , found'
+  , gatewayTimeout
+  , gatewayTimeout'
+  , gone
+  , gone'
+  , hTTPVersionNotSupported
+  , hTTPVersionNotSupported'
+  -- 3xx
```

Comment:
These are now sorted alphabetically and not by status code, so we should delete these comments

Reply:
Or, maybe we should disable formatting for this block, given that sorting by status code here might actually make more sense?  IDK, I don't have strong opinions, and I don't even know if disabling the formatter for a block is possible.  Thoughts?

Reply:
It’s not possible to disable the formatter, but I can disable sorting for the imports and exports from a module. (And revert the formatting commit.)

Reply:
No, I think the sorting is good in every other place than this, and sorting by status code here is arguably not great anyways since it's not enforceable in tooling.  Let's keep it.